project2-beat-that/script.js

```diff
@@ -1,4 +1,65 @@
+//for main function
+var player_score1 = 0;
+var player_score2 = 0;
+var playerFinalScore = 0;
+
+//players current score
+var player1_score = 0;
+var player2_score = 0;
+
+// players total score
+var player1_total = 0;
+var player2_total = 0;
+
+// counters
+var counter = 0;
+var remainder = 0;
+var player2 = 0;
+
+// arrray for score recording
+var allPlayersTotal = [0, 0];
+
+var rollDice = function () {
+  //player 1 first roll
+  var randomRollDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomRollDecimal) + 1;
+  return randomInteger;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  player_score1 = rollDice();
+  player_score2 = rollDice();
+
+  if (player_score1 < player_score2) {
+    playerFinalScore = player_score2 * 10 + player_score1;
+  } else {
+    playerFinalScore = player_score1 * 10 + player_score2;
+  }
```

Comment:
good, but not very scalable esp if you were to implement multiple dice, which is one of the more comfortable game modes. 

hint: can we store these dice rolls into an array, sort them in descending order, and then join the results back into a string/number?

see here for more details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort